### ISSUE TITLE
Measurement Annotations Too Far Off

### ISSUE THREAD

**Issue by Rionmonster** [chafey/cornerstoneTools#60]
I've found after a bit of testing that the annotations that appear with each of the major measurements (e.g. Length, Angles, etc.) appear to be quite far off from the actual areas that are being measured. This can often cause the measurements to be left off the canvas entirely as seen below : 

![Example of Incorrect Annotations](http://i.imgur.com/hTs2ey4.gif)

When troubleshooting the issue, I found that each of the X,Y coordinates seemed to usually be off by about ~1000 pixels and when explicitly subtracting those values : 

```
// Adjustment for annotations (Line 3053 of cornerstoneTools.js)
context.fillText(text, textX - 1000, textY - 1000);
```
After doing this the annotations usually appeared at a much more reasonable location : 

![Attempted Fix at Annotations](http://i.imgur.com/hTs2ey4.gif)

I know that this is by no means an attempt to fix or resolve this issue (or an appropriate way to handle it I believe). Something is likely wrong with how these values are being calculated. I noticed the issue first appeared after the latest release of the core library (cornerstone.js). It's likely a result of the `fontParameters.fontScale' variable being often quite small (~0.1) which could be causing the midpoint values to balloon up :

```
var textX = (data.handles.start.x + data.handles.end.x) / 2 / fontParameters.fontScale;
var textY = (data.handles.start.y + data.handles.end.y) / 2 / fontParameters.fontScale;
```

This may simply be something minor in the core that isn't being read properly or a variable that isn't being set, but I thought it was worth bringing up.

**Comment by swederik** [comment]
See https://github.com/chafey/cornerstoneTools/pull/59

I am in the process of updating all the tools to handle the new method for drawing the tools. The demos are just odd because they're relying on the newest version of cornerstone core. Use a slightly older version than master and the issue will go away.

**Comment by Rionmonster** [comment]
Good to hear Eric,

Previously I had elected to use an older version to resolve this, but thought I would ask anyways. 

I figured it was just an issue with the tools not yet catching up to the core.

**Comment by Rionmonster** [comment]
Good to hear Eric,

Previously I had elected to use an older version to resolve this, but thought I would ask anyways. 

I figured it was just an issue with the tools not yet catching up to the core. I'll just use the previously mentioned "hackish subtraction" approach in the mean-time until the fix is made (for testing things out at least).

**Comment by swederik** [comment]
Closing since I merged #59
